src/main/java/uk/gov/beis/els/api/categories/domesticovens/DomesticOvenInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.domesticovens;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotBlank;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.BaseInternetLabelApiForm;
+import uk.gov.beis.els.categories.domesticovens.service.DomesticOvensService;
+
+@Schema(name = "Domestic oven internet label")
```

Comment:
We call these 'internet labels' internally, but in the API doco I think we should call them 'arrow images' (eg "Domestic oven arrow image").

Avoiding the reference to 'internet' means we can use the same phrasing for old labels (where the arrow is just for websites) and new labels (where it's for any distance selling, eg TV adverts). Calling them images rather than label helps make it clear that it's not a full label.

Reply:
Have updated this in all user facing strings